src/Draft/Draft.js

```diff
@@ -1,63 +1,46 @@
 import React, { Component } from 'react'
 import { Subscription } from '../streamLib/stream.js'
+import { withUserContext } from '../Contexts/UserContext'
 import MediaQuery from 'react-responsive'
-
-import { UserContext } from '../App'
 import MyTeam from './MyTeam/MyTeam'
 import PlayerList from './PlayerList/PlayerList'
 import TeamList from './TeamList/TeamList'
 import CurrentBid from './PlayerList/CurrentBid'
 
 class Draft extends Component {
-
-  constructor (props) {
+  constructor(props) {
     super(props)
-    this.state = {teams: [], players: [], myTeam: null}
+    this.state = { teams: [], players: [], myTeam: null }
   }
 
   componentDidMount() {
-    this.subscription = new Subscription('/draft',
-    (data) => {
+    this.subscription = new Subscription('/draft', data => {
       this.setState({
-        myTeam: data.teams.find((team) => {
-          return team.roster.find((p)=> p._key === this.props.user._key)
+        myTeam: data.teams.find(team => {
+          return team.roster.find(p => p._key === this.props.user._key)
         }),
         teams: data.teams,
-        players: data.players})
+        players: data.players
+      })
     })
   }
 
   componentWillUnmount() {
     this.subscription && this.subscription.end()
   }
 
-  render () {
+  render() {
     return (
-      <div style={{color: 'white'}}>
-          {this.state.myTeam ?
-            <MyTeam
-              user={this.props.user}
-              team={this.state.myTeam} /> : ''}
-          <MediaQuery minDeviceWidth={1224}>
-            <TeamList
-              user={this.props.user}
-              teams={this.state.teams} />
-          </MediaQuery>
-          <CurrentBid
-            user={this.props.user} />
-          <PlayerList
-            user={this.props.user}
-            players={this.state.players} />
+      <div style={{ color: 'white' }}>
+        {this.state.myTeam ? <MyTeam user={this.props.user} team={this.state.myTeam} /> : ''}
+        <MediaQuery minDeviceWidth={1224}>
+          <TeamList user={this.props.user} teams={this.state.teams} />
+        </MediaQuery>
+        <CurrentBid user={this.props.user} />
+        <PlayerList user={this.props.user} players={this.state.players} />
```

Comment:
I don't see the purpose of one lining these unless its part of a standard style you can reference.

Reply:
The self enclosing tags? They aren't declaring any children here, so I don't think they need a closing tag. (not arguing, just defending)

Reply:
no I mean that props are all on one line